devkita64/Dockerfile

```diff
@@ -4,5 +4,7 @@ MAINTAINER Dave Murphy <davem@devkitpro.org>
 
 RUN dkp-pacman -Syyu --noconfirm switch-dev && \
     dkp-pacman -S --needed --noconfirm `dkp-pacman -Slq dkp-libs | grep '^switch-'` && \
-    dkp-pacman -Scc --noconfirm
+    dkp-pacman -Scc --noconfirm && \
+    dkp-pacman -S --needed --noconfirm devkitpro-pkgbuild-helpers
 
+ENV PATH=$DEVKITPRO/devkitA64/bin:$PATH
```

Comment:
devkitA64 should not be in global path. If we do this it then encourages sloppy projects that will cause problems for users. We've already seen this happen with people having multiple arm-none-eabi toolchains installed and attempting to use the wrong one for the wrong project.

Reply:
How to make it work with switch.cmake then?

Reply:
No. The point of switch.cmake is to help compiling ports which require cmake. The point of our Makefile based templates is to allow users to just run make & not care about any setup before building a project.

Currently we're sourcing switchvars.sh before running cmake, like this -> https://github.com/devkitPro/pacman-packages/blob/9128546582338c06d4fc11160cc04081823bc310/switch/bulletphysics/PKGBUILD#L20

There is some more discussion on updating the cmake support files here - https://github.com/devkitPro/pacman-packages/pull/64

Like I said on IRC, this is all a WiP and many things still need to be done despite the many things that have already been done.

Reply:
Ok, understood.